README.rst

```diff
@@ -29,6 +29,11 @@ Quick Links
 -----------
 
 * `Read the docs <https://falcon.readthedocs.io/en/stable>`_
+
+  `Getting help <https://falcon.readthedocs.io/en/stable/community/help.html>`_ -
+  `FAQ <https://falcon.readthedocs.io/en/stable/user/faq.html>`_ -
+  `Framework reference <https://falcon.readthedocs.io/en/stable/api/index.html>`_
```

Comment:
can't we make them render in new lines? I think they would look better. In markdown two spaces before a newline will break the line, not sure if it's the same with rst

Reply:
I'll try making this even more compact, inside parentheses on the same line, and will float on Gitter for evaluation.